lib/web_ui/test/felt_config.yaml

```diff
@@ -0,0 +1,268 @@
+compile-configs:
```

Comment:
Consider documenting at the top of the file common update scenarios, such as adding a new test suite. Or do we expect this to mostly stay as is?

Reply:
I do expect this to possibly change in the future. I think rather than doing a bunch of comments in the .yaml file, I'll just refer to the README.md in this directory which describes the structure of this file. Do you feel like that readme already has enough info? Or is there something more specific you'd like to see in there? I do have examples for each data type (test set, test bundle, test suite, run configuration, compile configuration, etc)